### PR TITLE
[DOCFIX] Improve Framework docs

### PR DESCRIPTION
**Pull request by calvinjia**
Clarifies how to run with Spark and removes a duplicate page.

### CODE REVIEW

File: docs/en/Running-Spark-on-Alluxio.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Alluxio works together with Spark 1.1 or later out-of-the-box.
 
 * Alluxio cluster has been set up in accordance to these guides for either
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+
+* Alluxio client will need to be compiled with the Spark specific profile. Build the entire project
+from the top level alluxio directory with the following command:
```
Comment (jsimsa) [inline docs/en/Running-Spark-on-Alluxio.md]: should we be using Alluxio or alluxio?

Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: or you can just code format it, or add the trailing slash.

**Comment by gpang** [comment]
I left a minor comment. Thanks!

File: docs/en/Running-Spark-on-Alluxio.md (hunk 1 above)
Comment (calvinjia) [inline docs/en/Running-Spark-on-Alluxio.md]: Good point, thanks.

**Comment by jsimsa** [comment]
LGTM

**Comment by apc999** [comment]
LGTM

**Pull request merged**